aten/src/ATen/native/quantized/cpu/qnnpack_fc.cpp

```diff
@@ -95,8 +96,13 @@ class QNNPACKLinear final : public torch::OperatorKernel {
     TORCH_INTERNAL_ASSERT(
         setupStatus == qnnp_status_success,
         "failed to setup QNNPACK Linear operator");
+    pthreadpool_t threadpool = nullptr;
+#ifdef C10_MOBILE
```

Comment:
can we move this #ifdef to the implementation side instead? return null there if it's not mobile build.